charts/core/src/XAxis/XAxis.tsx

```diff
@@ -87,17 +84,17 @@ const getOptions = ({
  *   <XAxis
  *     type="time",
  *     label="My X-Axis Data",
- *     unit="GB"
+ *     formatter="{value}GB"
```

Comment:
is this correct?

Reply:
Yup, exactly. Specifically for rendering the value. There are other formatters such as `'{yyyy}-{MM}-{dd}'` as well.